### PR TITLE
Restful api

### LINKED ISSUE #42: create Parkapp RESTful API
so the client can easily manage their data

### PR DESCRIPTION
**Pull request by sophiavocado**
closes #42

### CODE REVIEW

File: server/router.js [hunk 1]
```diff
@@ -1,6 +1,79 @@
 const express = require('express');
+// require('./db/db_connections')();
```
Comment (idantb101) [inline server/router.js]: Remind me please to come back to this bit together.

File: server/router.js [hunk 2]
```diff
@@ -1,6 +1,79 @@
...
+    park.tags = [req.body.tags[0], req.body.tags[1], req.body.tags[2]];
+    park.picturesUrl = {
+      small: [req.body.picturesUrl.small[0], req.body.picturesUrl.small[1]],
+      big: req.body.picturesUrl.big
+    };
+
+    park.save(err => {
+      if (err) {
+        console.log('this is error!', err);
+        res.json({ result: 0 });
```
Comment (idantb101) [inline server/router.js]: Shouldn't we also return here 500 >? as in line 10 ?

File: server/server.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 const express = require('express');
 const app = express();
 const path = require('path');
+const Park = require('./db/models/Park');
```
Comment (idantb101) [inline server/server.js:4]: Why is park upper case ?

File: server/server.js [hunk 4]
```diff
@@ -12,7 +13,7 @@ app.use(bodyParser.json());
 const morgan = require('morgan');
 app.use(morgan('dev'));
 
-const router = require('./router')();
+const router = require('./router')(app, Park);
```
Comment (idantb101) [inline server/server.js:16]: Why don't you just require the park inside the router?

**Review by idantb101 — COMMENTED** [review]
Generally looks amazing !!! 
please notice my small notes :+1:

**Review by NickP123 — APPROVED** [review]

File: server/server.js (hunk 4 above)
Comment (NickP123) [reply server/server.js:16]: I've updated all lang to long for longitude

**Comment by NickP123** [comment]
I've resolved the conflicts  - were just spelling issues.

**Pull request merged**